src/App.tsx

```diff
@@ -616,6 +632,88 @@ class App extends React.Component<Props, State> {
                 component={AdminPage}
             />
 
+            <PrivateRoute
+                auth={this.state.auth}
+                path="/MyAccount"
+                component={MyAccountPage}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources"
+              exact={true}
+              component={CoachingResources}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/CoachingCycle"
+              exact={true}
+              component={CoachingCoachingCycle}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials"
+              exact={true}
+              component={CoachingProfessionalDevelopmentMaterials}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/TransitionTime"
+              component={CoachingTransitionTime}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/ClassroomClimate"
+              component={CoachingClassroomClimate}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/MathInstruction"
+              component={CoachingMathInstruction}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/LevelOfInstruction"
+              component={CoachingLevelOfInstruction}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/StudentEngagement"
+              component={CoachingStudentEngagement}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/ListeningToChildren"
+              component={CoachingListeningToChildren}
+            />
+            <PrivateRoute
+              auth={this.state.auth}
+              path="/CoachingResources/ProfessionalDevelopmentMaterials/SequentialActivities"
+              component={CoachingSequentialActivities}
+            />
+            {/*
+            <PrivateRoute
```

Comment:
@thompchr I saw a comment from you that you'd added the literacy pdf thing, so I think maybe this needs to be uncommented?